src/com/tfl/billing/JourneyCostCalculator.java

```diff
@@ -0,0 +1,77 @@
+package com.tfl.billing;
+
+import com.tfl.external.Customer;
+import com.tfl.external.PaymentsSystem;
+import java.math.BigDecimal;
+import java.util.*;
+
+
+public class JourneyCostCalculator implements CostManager {
+
+    private static final BigDecimal PEAK_LONG_JOURNEY_PRICE = new BigDecimal(3.80);
+    private static final BigDecimal PEAK_SHORT_JOURNEY_PRICE = new BigDecimal(2.90);
+    private static final BigDecimal OFF_PEAK_LONG_JOURNEY_PRICE = new BigDecimal(2.70);
+    private static final BigDecimal OFF_PEAK_SHORT_JOURNEY_PRICE = new BigDecimal(1.60);
+
+    BigDecimal getTotalFromJourneyList(List<Journey> journeys, BigDecimal customerTotal) {
```

Comment:
use protected instead. This will not have any subclasses anyways: https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html